src/rich-text-editor.component.js

```diff
@@ -19,11 +19,16 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   const [hasSetInitialHTML, setHasSetInitialHTML] = useState(false);
 
   if (editorRef) {
-    editorRef.current = {
+    const current = {
       setHTML,
       resetEditor,
       getHTML,
       focus,
+    };
+    if (editorRef instanceof Function) {
```

Comment:
Using `instanceof` for primitives is often a bit scary ([stackoverflow](https://stackoverflow.com/questions/899574/what-is-the-difference-between-typeof-and-instanceof-and-when-should-one-be-used)). Looks like it's okay to use with functions, but in general using typeof for primitives is better.

Since there is no bug here, I've still approved the PR.

```suggestion
    if (typeof editorRef === 'function') {
```

Reply:
fixed 👍